spec.html

```diff
@@ -25358,10 +25361,10 @@ <h1>Number.isSafeInteger ( _number_ )</h1>
 
       <emu-clause id="sec-number.max_safe_integer">
         <h1>Number.MAX_SAFE_INTEGER</h1>
+        <p>The value of Number.MAX_SAFE_INTEGER is 9007199254740991 (2<sup>53</sup> - 1).</p>
         <emu-note>
-          <p>The value of `Number.MAX_SAFE_INTEGER` is the largest integer n such that n and n + 1 are both exactly representable as a Number value.</p>
+          <p>Integers greater than `Number.MAX_SAFE_INTEGER` cannot be uniquely represented by Number values and are therefore not necessarily distinguishable. For example, both `9007199254740992` and `9007199254740993` evaluate to the Number value 9007199254740992.</p>
```

Comment:
I will concede that the previous explanation was more **clear**, and am open to feedback on how to recover that. But the two are equally **correct** (responding directly to your point, the integer 9007199254740992 is not uniquely represented by the Number value 9007199254740992, because the distinct integer 9007199254740993 is also represented by the same Number value—that you initially misunderstood the use of "uniquely represented" is an issue of clarity, not correctness), and the previous explanation is _less_ **useful** (because it fails to convey why anyone should care about n + 1 as opposed to e.g. n + 2, n + 10, or n − 1).

Reply:
> the integer 9007199254740992 is not uniquely represented by the Number value 9007199254740992, because the distinct integer 9007199254740993 is also represented by the same Number value

I don't think the spec supports this phrasing. The Number value 9007199254740992 *represents* the integer 9007199254740992 and *only* that integer.

Instead, it sounds like the many-to-one relation you have in mind is the one defined for the phrase "the Number value of x". (I.e., every [non-NaN] Number value is the Number value of infinitely many mathematical values x.)

Reply:
@jmdyck that text no longer appears in this PR.